tests/subsys/event_manager_proxy/common_events/data_events.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _DATA_EVENTS_H_
+#define _DATA_EVENTS_H_
+
+/**
+ * @brief Data Event
+ * @defgroup data_event Data Event
+ * @{
+ */
+
+#include <app_event_manager.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define DATA_BIG_EVENT_BLOCK_SIZE 64
+
+
+struct data_event {
+	struct app_event_header header;
```

Comment:
Add documentation for consistency  as you add it in test_events.h

Reply:
Ok.